src/test/resources/projects/maven/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/annotations/PostConstructAnnotation.java

```diff
@@ -8,25 +8,9 @@ public class PostConstructAnnotation {
 
     private Integer studentId;
 
-    private boolean isHappy;
```

Comment:
This class is part of the sample application which was written for LSP4Jakarta. I believe this code was pulled in from LSP4Jakarta without any changes and should probably be extracted into its own project one day so that it can be shared between Eclipse, IntelliJ and any other one our tools that requires the Jakarta EE sample for testing. It's unclear to me why this is being changed in this PR, in particular why some of the code has been deleted. Can you provide an explanation?